website/docs/workloads/blender/blender.md

```diff
@@ -0,0 +1,57 @@
+# Blender Benchmark
+Blender Benchmark, a new platform to collect and display the results of hardware and software performance tests. With this benchmark we aim at an optimal comparison between system hardware and installations, and to assist developers to track performance during Blender development.
+
+The benchmark consists of two parts: a downloadable package which runs Blender and renders on several production files, and the Open Data portal on blender.org, where the results will be (optionally) uploaded.
+
+* [Blender benchmark] (https://www.blender.org/news/introducing-blender-benchmark/)
+
+## What is Being Measured 
```

Comment:
Question: is Blender and Blender-benchmark separate things? Could you document how this works in bullet point 1,2,3?

Reply:
blender-benchmark tests the rendering speed using blender. will make the doc more clear.